.gitlab/spack_packages/sundials/package.py

```diff
@@ -908,6 +942,9 @@ def initconfig_package_entries(self):
         if "+trilinos" in spec:
             entries.append(cmake_cache_path("Trilinos_DIR", spec["trilinos"].prefix))
 
+        if "+profile-examples" in spec:
+            entries.append(cmake_cache_path("SUNDIALS_CALIPER_OUTPUT_DIR"), "/usr/workspace/sundials/califiles")
```

Comment:
Maybe we can make this a variant that takes a value and the value is the `SUNDIALS_CALIPER_OUTPUT_DIR`. This way the path is not hardcoded in to the package.py.